archive/frames/models.py

```diff
@@ -154,7 +154,7 @@ def as_dict(self):
 
         if self.area:
             ret_dict['area'] = json.loads(self.area.geojson)
-        ret_dict['related_frames'] = [rf.id for rf in self.related_frames.all()]
+        ret_dict['related_frames'] = list(self.related_frames.all().values_list('id', flat=True))
```

Comment:
isn't it already a list if Flat=True? is casting it to a list() needed for some corner case?